src/renderers.tsx

```diff
@@ -283,8 +283,9 @@ export function createRenderers(opts: Required<Option>, ref: any, store: Compone
           </tr>
         );
       case opts.typeCell:
+        const style = props.node.data.get('style') || {};
         return (
-          <Cell editor={editor} store={store} node={props.node} attributes={props.attributes} opts={opts}>
+          <Cell editor={editor} store={store} node={props.node} attributes={props.attributes} opts={opts} style={style}>
```

Comment:
@prpateldev Thanks for your PR. I guess we need to add `style` to CellProps.

Reply:
I am going to make new pull request.
I have extract style in the Cell Component